packages/components/src/content/ScanQrCode/index.tsx

```diff
@@ -13,10 +13,26 @@ export type IScanQrCodeProps = {
 };
 
 export function ScanQrCode({ handleBarCodeScanned }: IScanQrCodeProps) {
+  const scanned = useRef(false);
   const [currentPermission, setCurrentPermission] = useState<PermissionStatus>(
     PermissionStatus.UNDETERMINED,
   );
   const isFocused = useIsFocused();
```

Comment:
```suggestion
  const isFocused = useIsFocused();
  if (isFocused) {
      scanned.current = false;
    }
```
ref 的更新，不需要用 useEffect

Reply:
我在另外的 PR 解决这里哈

Reply:
@sidmorizon 这这里哈 https://github.com/OneKeyHQ/app-monorepo/pull/4096/commits/29d50bb0d5aa3de256843b1531ba0650ccd74385